site/package.json

```diff
@@ -22,10 +22,14 @@
         "@types/node": "18.11.18",
         "@types/react": "18.0.26",
         "@types/react-dom": "18.0.10",
+        "cookie": "^0.5.0",
```

Comment:
Do we need this as a dependency?

Reply:
yes for this import { CookieSerializeOptions } from "cookie";